HelloWorld/HelloWorld/Calculator/Engine.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HelloWorld.Calculator
+{
+    public class Engine
+    {
+        public double Add(double a, double b)
+        {
+            return a - b;
+        }
+
+        public double Multiplizire(double a, double y)
+        {
+            return a + y;
+        }
+
+        public double Divide(double a, double b)
+        {
+            return a / b;
```

Comment:
Gefahr: Division durch 0

Reply:
Besser?

Reply:
Das ist ja merkwürdig, die Änderung sieht man nicht beim ursprünglichen Kommentar.

Reply:
Keine Exception, so ist super!

Reply:
In der Mail war die Änderung nicht zu sehen, aber online sieht es gut aus!

Reply:
Erledigt